srcpkgs/glslViewer/template

```diff
@@ -1,31 +1,28 @@
 # Template file for 'glslViewer'
 pkgname=glslViewer
-version=1.6.8
+version=3.0.7
 revision=1
-build_style=gnu-makefile
-make_use_env=yes
+_vera_gitrev="91c48874900909ebd58440359657948aed906f50"
+build_style=cmake
 hostmakedepends="pkg-config"
 makedepends="glfw-devel glu-devel MesaLib-devel libX11-devel libXrandr-devel
  libXi-devel libXxf86vm-devel libXcursor-devel libXinerama-devel libXext-devel
- libXrender-devel libXdamage-devel"
+ libXrender-devel libXdamage-devel ncurses-devel libatomic-devel liblo-devel"
 short_desc="Live GLSL coding render"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="http://patriciogonzalezvivo.com/2015/glslViewer/"
-distfiles="https://github.com/patriciogonzalezvivo/glslViewer/archive/${version}.tar.gz"
-checksum=3792984457c9487cd1f87374b7698f3b66f4480b3eeac7d940a2ecf9e74dee3e
+distfiles="https://github.com/patriciogonzalezvivo/glslViewer/archive/${version}.tar.gz
+ https://github.com/patriciogonzalezvivo/vera/archive/${_vera_gitrev}.tar.gz"
+checksum="4d8bac6a801fa61b81e472a8dabfe5469a43c44547d558849a078ed2c26c8034
+ be12b5876bcea771237e36ad9b3ea9ebc3f1fa6cc1a7e25a5cd4f5988210e041"
+skip_extraction="${_vera_gitrev}.tar.gz"
```

Comment:
can we not skip extraction and instead use `build_wrksrc=${pkgname}-${version}` and `mv ${wrksrc}/vera to ${wrksrc}/deps/vera` in post_extract?

Reply:
Implemented your suggestion.